application/src/main/resources/thingsboard.yml

```diff
@@ -722,10 +734,18 @@ spring.jpa.properties.hibernate.dialect: "${SPRING_JPA_DIALECT:org.thingsboard.s
 
 # SQL DAO Configuration parameters
 spring:
+  autoconfigure:
+    # Exclude Spring Boot auto-configuration classes that you do not want to apply to your application
+    exclude: "${SPRING_AUTOCONFIGURE_EXCLUDE: org.springframework.boot.autoconfigure.mongo.MongoAutoConfiguration}"
```

Comment:
Buna neden gerek oluyor?

Reply:
Oto konfigürasyonu önceden buradan devre dışı bırakmışlar, biz de bunu direk kopyaladık.
Oto konfigürasyonun devre dışı bırakılmasının sebebi ise mongoDb'de 2 adet dokümana (reading type, transformation) URI'lar ile bağlandığımız için otomatik konfigürasyonu kullanmak yerine hangi mongoDb repo'sunun hangi URI'ını kendimiz vermemiz gerekiyor.

Reply:
Bu ezbere eklenmiş galiba kaldırıp test ettim değişiklik olmadı